src/researches/keywordCount.js

```diff
@@ -12,8 +11,12 @@ const escapeRegExp = require( "lodash/escapeRegExp" );
  * @returns {number} The keyword count.
  */
 module.exports = function( paper ) {
-	const keyword = escapeRegExp( normalizeQuotes( paper.getKeyword() ) );
+	const keywordForms = buildKeywordForms( paper );
```

Comment:
`normalizeQuotes`  was removed for the keyword, but it is still used for the text. So now when you have a keyword with non-normalized quotes, e.g., `brain’s`,  it won't be found in the text because the normalized version of that would be `brain's`.

I was also looking at this issue: https://github.com/Yoast/YoastSEO.js/issues/1399 The idea is that we should always normalize the quotes for the assessments. I wonder whether there is a more central location where we can take care of it, rather than in each individual assessment. We should have a look at this.

Reply:
Thanks for pointing it out! I took care of these cases in my other PR #1536 that processes possessive forms.